db/schema.rb

```diff
@@ -27,14 +27,15 @@
   add_index "procurement_areas", ["locations"], name: "index_procurement_areas_on_locations", using: :gin
   add_index "procurement_areas", ["memberships"], name: "index_procurement_areas_on_memberships", using: :gin
 
-  create_table "users", force: :cascade do |t|
-    t.integer  "defence_request_uid"
-    t.integer  "defence_request_access_token"
-    t.string   "email"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+  create_table "shifts", force: :cascade do |t|
+    t.string   "name"
+    t.string   "location_uid",  null: false
+    t.time     "starting_time"
+    t.time     "ending_time"
```

Comment:
After speaking with @pixeltrix, it might be a good idea to store these as some sort of timestamp or number of seconds since the beginning of the day? Otherwise when we try and look up the slot that's for a given moment in time, we'll have to do some fudging of this field for computational purposes.